README.md

```diff
@@ -36,35 +35,31 @@ Table of contents
 - [Why “Arsenik”?](#why-arsenik)
 - [Join the community](#join-the-community)
 
-
 Philosophy
 --------------------------------------------------------------------------------
 
 **Bring the keys to your fingers, rather than moving your fingers to the keys.**
 
 Not sure if you should buy that expensive ergonomic keyboard?
 
-Download a ready-to-use Arsenik configuration for [Kanata], and enjoy on your
-regular keyboard features that were normally only accessible to programmable
+Download a ready-to-use Arsenik configuration for [Kanata], and enjoy your
+regular features that were normally only accessible to a programmable
 keyboard.
 
-*Note: You might benefit the most of Arsenik if you are [touch typing].*
+*Note: You probably will benefit the most of Arsenik if you are [touch typing].*
 
-
-Pick Your Poison!
+Pick Your Poison
```

Comment:
Scandalous ! This is some **very** exciting stuff, the exclamation mark is of the utmost importance !!